project/src/main/java/com/google/sps/data/UserCompetition.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class representing each competition object with user relevant info
+ */
+public class UserCompetition {
```

Comment:
Next class you write, can you switch to AutoValue. There are example from what Phoebe did.